crates/ruff/src/rules/ruff/snapshots/ruff__rules__ruff__tests__RUF200_bleach.snap

```diff
@@ -0,0 +1,17 @@
+---
+source: crates/ruff/src/rules/ruff/mod.rs
+---
+./resources/test/fixtures/ruff/pyproject_toml/bleach/pyproject.toml:5:16: RUF200 Failed to parse pyproject.toml: Version specifier `>=1.1.0<1.2` doesn't match PEP 440 rules
+tinycss2>=1.1.0<1.2
+        ^^^^^^^^^^^
```

Comment:
This breaks the normal style but unfortunately serde has no concept of spans so pep440_rs/pep508_rs can't forward this to toml; i think that's the best we can do